sql/parser/sql.y

```diff
@@ -861,14 +861,14 @@ iso_level:
     // Mapped to the closest supported isolation level.
     $$ = SnapshotIsolation
   }
-| REPEATABLE READ
+| SNAPSHOT
   {
-    // Mapped to the closest supported isolation level.
     $$ = SnapshotIsolation
   }
-| SNAPSHOT
+| REPEATABLE READ
   {
-    $$ = SnapshotIsolation
+    // Mapped to the closest supported isolation level.
```

Comment:
Depends on what the user expectation is. If they are asking for `REPEATABLE READ` in order to get better performance wouldn't it be better to say, err, no we can't do that. If they are asking for `REPEATABLE READ` in order to get certain guarantees then swapping in `SERIALIZABLE` makes more sense.